com.unity.render-pipelines.high-definition/Runtime/Lighting/ScreenSpaceLighting/ScreenSpaceGlobalIllumination.compute

```diff
@@ -135,16 +135,27 @@ void TraceReflectionProbes(PositionInputs posInput, float3 normalWS, float3 rayD
     GetCountAndStart(posInput, LIGHTCATEGORY_ENV, envLightStart, envLightCount);
     totalWeight = 0.0f;
 
+    uint envStartFirstLane;
```

Comment:
Force scalarization makes the kernel twice faster on ps4 as it is VGPR bound by the reflection probe code.